packages/block-library/package.json

```diff
@@ -27,8 +27,7 @@
 	"sideEffects": [
 		"build-style/**",
 		"src/**/*.scss",
-		"{src,build,build-module}/*/init.js",
-		"{src,build,build-module}/utils/interactivity/index.js"
```

Comment:
The same side effects weren't added to the new `@wordpress/interactivity` package. I might cause issues in the long run.

Reply:
Thanks, Greg. Added to the [roadmap](https://github.com/WordPress/gutenberg/discussions/52904) 🙂👍